e2e/multisig-migration.test.ts

```diff
@@ -107,6 +114,71 @@ describe("Should change LBC owner to the multisig.ts", function () {
     );
     await expect(newLbc.version()).to.eventually.be.equal("1.3.0");
   });
+
+  it("Should change the owner using Tenderly", async () => {
+    await checkForkedNetwork();
+    const deploymentNetwork = hre.network.name;
+    const multisigAddress = multsigInfo[deploymentNetwork].address;
+    if (!multisigAddress || multisigAddress === "")
+      throw new Error("Multisig address not found for current network");
+
+    // Get the current owner of LiquidityBridgeContract
+    const deploymentData = read()[deploymentNetwork];
+    const proxyAddress = deploymentData.LiquidityBridgeContract.address;
+    if (!proxyAddress)
+      throw new Error(
+        `LiquidityBridgeContract address not found on network ${deploymentNetwork}`
+      );
+
+    const contract = await ethers.getContractAt(
+      "LiquidityBridgeContractV2",
+      proxyAddress
+    );
+    const currentOwner = await contract.owner();
+    console.info(`Current owner: ${currentOwner}`);
+
+    // Impersonate the current owner account
+    await helpers.impersonateAccount(currentOwner);
+    const impersonatedSigner = await ethers.getSigner(currentOwner);
+
+    // Set balance for the impersonated account
+    const desiredBalance = ethers.toQuantity(ethers.parseEther("100"));
+    await ethers.provider.send("hardhat_setBalance", [
+      impersonatedSigner.address,
+      desiredBalance,
+    ]);
+
+    console.info(`Impersonated account: ${impersonatedSigner.address}`);
+
+    // Call changeMultisigOwner
+    await expect(
+      changeMultisigOwner(
+        multisigAddress,
+        deploymentNetwork,
+        impersonatedSigner
+      )
+    ).to.not.be.reverted;
```

Comment:
This is not a transaction, is a script that creates two transaction its result is a `Promise<void>`, I'm not sure how that behaves with the `await expect().not.to.be.reverted` I propose to change it for `await expect().resolves.not.toThrow()`